test/cri-containerd/runpodsandbox_test.go

```diff
@@ -1062,7 +1063,7 @@ func createExt4VHD(ctx context.Context, t *testing.T, path string) {
 		log.L.Logger.SetOutput(io.Discard)
 		defer log.L.Logger.SetOutput(origLogOut)
 	}
-	uvm := testutilities.CreateLCOWUVM(ctx, t, t.Name()+"-createExt4VHD")
+	uvm := testuvm.CreateAndStartLCOW(ctx, t, t.Name()+"-createExt4VHD")
```

Comment:
Youre right, `t.Fatal` wont stop a test when called from a go routine, but it will kill that go routine (and set `t.Failed` to true)
all these mostly just wrap some internal function, but add an `if err != nil { t.Fatalf(err) }`, since thats the most common case and I am tired of always writing that out 😅....

I think we do need some testing utilities, and will probably refactor this code to use those, but I would rather save that for a separate PR and more discussion

Reply:
Sounds good.